KsApi/ServiceType.swift

```diff
@@ -73,6 +73,10 @@ public protocol ServiceType {
 
   func createBacking(input: CreateBackingInput) ->
     SignalProducer<CreateBackingEnvelope, ErrorEnvelope>
+  
+  /// Sends report project data to relevant to a specific project
```

Comment:
nit: This sentence doesn't make sense to me. Maybe just "Sends report project data for a specific project"?